config/environments/local.rb

```diff
@@ -0,0 +1,43 @@
+MyLibraryNYC::Application.configure do
+  # Settings specified here will take precedence over those in config/application.rb
+
+  # In the development environment your application's code is reloaded on
+  # every request. This slows down response time but is perfect for development
+  # since you don't have to restart the web server when you make code changes.
+  config.cache_classes = false
+  # config.cache_store = :memory_store
+
+  # Log error messages when you accidentally call methods on nil.
+  config.whiny_nils = true
+
+  # Show full error reports and disable caching
+  config.consider_all_requests_local       = true
+  config.action_controller.perform_caching = false
+
+  # Don't care if the mailer can't send
+  config.action_mailer.raise_delivery_errors = true
```

Comment:
Does the doc string correspond to the true value?

Reply:
I'll remove it to be safe.  It's ambiguous.